docs/site/guides/rest/rest-api-guides.md

```diff
@@ -0,0 +1,7 @@
+---
+lang: en
+title: 'REST API Guides'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, REST
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/rest-api-guides.html
+---
```

Comment:
Let's remove this empty page after we land https://github.com/strongloop/loopback.io/pull/996